about/pricing-and-account-management.md

```diff
@@ -61,21 +61,23 @@ from initial development through to mission-critical enterprise applications.
    or egress. There are no per-query fees, nor additional costs to read or write data. It's all completely 
    transparent, easily understood, and up to you.
   
-### $CLOUD_LONG free trial for the different price plans
+### Test $CLOUD_LONG for free
 
-We offer new users a free, 30-day trial period of our $PERFORMANCE plan with no credit card required.  
-During your trial, you can contact $CONTACT_SALES to request information about, and access
-to, our $SCALE plan to determine how it fits your needs. During your trial, if a $SERVICE_SHORT doesn’t receive any queries for 7 days, it is paused to conserve resources. Your data remains intact during the trial, and you can easily resume your $SERVICE_SHORT in $CONSOLE. After your trial ends, we may remove your data unless you’ve added a payment method.
+Are you just starting out with $CLOUD_LONG? On our Free pricing plan, you can create $SERVICE_SHORTs with fixed limited CPU, memory, and storage resources, and run them at zero cost. 
```

Comment:
```suggestion
Are you just starting out with $CLOUD_LONG? On our Free pricing plan, you can create up to two zero cost $SERVICE_SHORTs with shared compute and 750 MiB storage. 
```
We should be explicit here about the specs.  Maybe a variable for the amount of storage, so it can be easily changed if we need to